src/resources/general/performance-analysis.md

```diff
@@ -0,0 +1,28 @@
+# Low Level Analysis Resources
+
+## Profiling
```

Comment:
Missing a mention of VS's built in tooling for sampling and instrumented performance analysis